components/docs-chef-io/content/habitat/get_started.md

```diff
@@ -0,0 +1,257 @@
+- The [Chef Habitat CLI]({{< relref "/habitat/install_habitat" >}}) installed locally
+- An account on [Chef Habitat Builder]({{< relref "builder_account" >}})
+- A [profile on you Builder account]({{< relref "builder_profile" >}})
+
+### Create an origin and set up the Habitat CLI
+
+1. If you don't already have one, [create an origin]({{< relref "habitat/builder_origins" >}}) in your Chef Habitat Builder account. This origin will store the Habitat package that you create in this tutorial.
+
+1. If you haven't already done so, run `hab cli setup` and follow the instructions. This will configure the Hab CLI to work with your origin on the Habitat Builder SaaS.
+
+   You will need to provide the origin you created in your Builder account and a personal access token, which you can generate in your [Profile page in Habitat Builder](https://bldr.habitat.sh/#/profile).
+
+## Fork and Clone the Habitat sample-node-app repository
+
+1. Create a fork of the [sample-node-app](https://github.com/habitat-sh/sample-node-app) GitHub repository in your own GitHub account.
+
+1. Clone the fork onto your local workstation.
+
+## Edit the Plan File
+
+The [plan file]({{< relref "/habitat/plan_writing" >}}) instructs Chef Habitat in how to build, deploy, and manage your application.
+
+Edit the plan file to user your origin:
+
+1. Open the `sample-node-app/habitat/plan.sh` file with your editor of choice.
+1. Update the value of `pkg_origin` with the name of the origin in your Habitat Builder account.
+
+Note the `pkg_deps` setting. This lists dependencies that your package requires at runtime. This Node.js application requires the [`core/node` package](https://bldr.habitat.sh/#/pkgs/core/node/latest), which is maintained by the Habitat Core maintainers. You can see a full list of core packages in the [core-plans](https://github.com/habitat-sh/core-plans) repository, or by searching for **core** in Habitat Builder.
+
+See the [plan settings]({{< relref "/habitat/plan_settings" >}}) documentation for more information about the settings in a plan file.
+
+See the [plan writing]({{< relref "/habitat/plan_writing#define-your-dependencies" >}}) documentation for more information defining dependencies.
+
+## Run the Service in the Habitat Studio
+
+{{< readfile file="content/habitat/reusable/md/habitat_studio_overview.md" >}}
+
+Use the Habitat Studio to generate a Habitat package and preview the running Node.js app:
+
+1. In a terminal, switch to the root of the `sample-node-app` directory.
+
+1. If you're using Docker, expose port 8000 so you can preview the application in a browser.
+
+   To expose port 8000 on a macOS or Linux workstation, run:
+
+   ```bash
+   export HAB_DOCKER_OPTS="-p 8000:8000"
+   ```
+
+   To expose port 8000 on a Windows workstation, run:
+
+   ```bash
+   $env:HAB_DOCKER_OPTS="-p 8000:8000"
+   ```
+
+1. Enter the Habitat Studio:
+
+   ```bash
+   hab studio enter
```

Comment:
note this will only enter a docker studio on a mac. You will need to add `-D` on linux or windows.